cluster.tf

```diff
@@ -27,17 +38,24 @@ resource "google_container_cluster" "primary" {
   resource_labels = {
     project     = var.project_id
     environment = var.environment
-    tenant      = "global"
+  }
+
+  # We should encrypt etcd cluster for security
+  database_encryption {
+    state    = "ENCRYPTED"
+    key_name = google_kms_crypto_key.encryption_key.self_link
   }
 
   master_authorized_networks_config {
+    # Let the master open to the world could be risky
     cidr_blocks {
       cidr_block   = "0.0.0.0/0"
       display_name = "All"
     }
   }
+
   lifecycle {
-    prevent_destroy = true
+    # prevent_destroy = true
```

Comment:
Should we revert this once before merging back to master?

Reply:
Yes, great catch thanks!